### ISSUE TITLE
Some blocks will be synchronized only after a restart！！！！

### ISSUE THREAD

**Issue by leezijie** [MinaProtocol/mina#11476]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/MinaProtocol/mina/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/MinaProtocol/mina/discussions

### Description

I started the node through docker, and the data was synchronized for a day, but the state was still catchup. Nothing changes in the block. But when I restart the container there will be some increase in height

### Steps to Reproduce

`1.docker run  -d -p 9934:3085 -p 9935:3087 -p 10101:10101 -v=/data/blockdata/mina/data:/data:rw --name mina minaprotocol/mina-rosetta:1.3.1-3e3abec-focal`
`2.mina client status`

### Expected Result

Normal data synchronization

### Actual Result

It's stuck. I don't know how long it takes

### How frequently do you see this issue?

Always

### What is the impact of this issue on your ability to run a node?

Blocker

### Status

```shell
Global number of accounts:                     115178
Block height:                                  155693
Max observed block height:                     155943
Max observed unvalidated block height:         155944
Local uptime:                                  1d9h24s
Ledger Merkle root:                            jxtyFR4kSXKQZFh2LmtaaedYMSfGFfVtmrzLNeyVd2rv3xNPBt1
Protocol state hash:                           3NKDSwXf3DbLxWmEfNPxLaUb2xGk3QWW2UjdTayH16Pr3yZ6PMwN
Chain id:                                      5f704cc0c82e0ed70e873f0893d7e06f148524e3f0bdae2afb02e7819a0c24d1
Git SHA-1:                                     3e3abecd4fd197017321d61a65a25f0bbdc40f3a
Configuration directory:                       /data/.mina-config
Peers:                                         15
User_commands sent:                            0
SNARK worker:                                  None
SNARK work fee:                                100000000
Sync status:                                   Catchup
Catchup status:
	To build breadcrumb:           1
	To initial validate:           0
	Finished:                      42
	To download:                   0
	Waiting for parent to finish:  87
	To verify:                     163

Block producers running:                       0
Coinbase receiver:                             Block producer
Best tip consensus time:                       epoch=32, slot=1097
Best tip global slot (across all hard-forks):  229577
Consensus time now:                            epoch=32, slot=1482
Consensus mechanism:                           proof_of_stake
Consensus configuration:
	Delta:                     0
	k:                         290
	Slots per epoch:           7140
	Slot duration:             3m
	Epoch duration:            14d21h
	Chain start timestamp:     2021-03-17 00:00:00.000000Z
	Acceptable network delay:  3m
```

**Comment by shimkiv** [comment]
@leezijie Thanks for filing an issue.
I'd like to know more about it though because I can't reproduce it on `GCP`, `Hetzner Cloud` and running it locally (using steps mentioned in ticket).
With enough resources (I tried with 8xCPU cores, 16Gb of RAM) it gets into `Synced` state all the time in about 1-1.5 hrs.
Can you please share more info about any possible specifics of environment in use on your end? Maybe you even have logs available since the issue was spotted?
Thank you.

**Comment by leezijie** [comment]
> @leezijie Thanks for filing an issue. I'd like to know more about it though because I can't reproduce it on `GCP`, `Hetzner Cloud` and running it locally (using steps mentioned in ticket). With enough resources (I tried with 8xCPU cores, 16Gb of RAM) it gets into `Synced` state all the time in about 1-1.5 hrs. Can you please share more info about any possible specifics of environment in use on your end? Maybe you even have logs available since the issue was spotted? Thank you.

Yes, my machine is fully equipped.After I restarted the node numerous times, it finally synced to the latest block.

**Comment by shimkiv** [comment]
@leezijie thank you for reply. 
Did you used any Cloud providers to run the server? Do you think there could have been the networking issues specific to environment in use?

**Comment by shimkiv** [comment]
Closing this issue because of being inactive for long time.
Please feel free to open another ticket should you experience any further issues.
